authors/apps/authentication/urls.py

```diff
@@ -1,12 +1,12 @@
-from django.conf.urls import url
 from django.urls import path
 
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
+
 app_name = "authentication"
 urlpatterns = [
-   path('user/', UserRetrieveUpdateAPIView.as_view()),
-   path('users/', RegistrationAPIView.as_view()),
-   path('users/login/', LoginAPIView.as_view()),
+    path('user/', UserRetrieveUpdateAPIView.as_view(), name="user_ru"),
```

Comment:
A little consistency would assist. Something like `name="user_retrieve_update"`. This also improves readability.

Reply:
Thanks for the clarification.